src/_includes/sm_cta_slice.html

```diff
@@ -1,6 +1,7 @@
 <div class="sm-cta-slice bg-coral text-white g-pt-10 g-pb-10">
   <div class="container">
     <svg class="sm-icon--cta" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 164.66 187.42"><defs></defs><g data-name="Layer 2"><g data-name="Layer 1"><path d="M143.85 59.65L84.01 94.52v89.9L22.9 152.27v-45.93l38.62 21.39v26.37l-19.48-10.25V73.42L61.52 85l43.29-25.35-42.52-24.92L22.9 58.29v23.76L3 71.89v-22.2L84.01 3l77.65 43.63v93.39l-38.22 22.2v-29.85"/><circle cx="124.13" cy="121.42" r="10"/></g></g></svg>
-    {% t software-modernisation.home-page-banner-text %} <a class="text-white" href="{{ '/services/software-modernisation/' | prepend: site.baseurl }}"><strong>{% t software-modernisation.home-page-banner-link %}</strong></a>
+    {% t software-modernisation.home-page-banner-text %}
+    {%- include soft_mod_husbpot_header_banner_cta.liquid -%}
```

Comment:
@mattgraygithub let's make note of this change as we will have conflicts to resolve on our branch where we cleaned up **sm_cta_slice**